ui/page/youtubeSync/view.jsx

```diff
@@ -69,7 +72,8 @@ export default function YoutubeSync(props: Props) {
       immediate_sync: true,
       desired_lbry_channel_name: `@${channel}`,
       return_url: `https://${DOMAIN}/$/${inSignUpFlow ? PAGES.AUTH : PAGES.YOUTUBE_SYNC}`,
-    }).then(ytAuthUrl => {
+      channel_language: language,
```

Comment:
This needs to be moved up 2 lines so it's passed to `yt/new`. It's not doing anything here.